packages/shared/src/components/UpgradeToPlus.tsx

```diff
@@ -57,13 +67,15 @@ export const UpgradeToPlus = ({
         tag="a"
         variant={ButtonVariant.Secondary}
         className={classNames(
-          'border-action-plus-default text-action-plus-default',
+          !color && 'border-action-plus-default text-action-plus-default',
```

Comment:
Shouldn't it be color ? color : 'colors'? (just checking they don't overlap

Reply:
Implemented logic is: If we pass the "color prop" we don't want to apply the linked classes, so we remove them

Reply:
Oh sorry misread this line actually, don't mind me :)